components/Match/matchColumns.jsx

```diff
@@ -249,42 +249,109 @@ const laningColumns = [
     field: 'lane_efficiency',
     width: 1,
     sortFn: true,
+    displayFn: (row, col, field) => field.toFixed(2),
   }, {
     displayName: 'LH@10',
     field: 'lh_t',
     width: 1,
     sortFn: true,
+    displayFn: (row, col, field) => field ? field[10] : '',
   }, {
     displayName: 'DN@10',
     field: 'dn_t',
     width: 1,
     sortFn: true,
+    displayFn: (row, col, field) => field ? field[10] : '',
   }];
 
+const purchaseColumns = [
+  heroTdColumn,
+  {
+    displayName: 'TP',
+    tooltip: 'Town Portal Scrolls purchased',
+    field: 'purchase',
+    width: 1,
+    sortFn: true,
+    displayFn: (row, col, field) => field ? field.tpscroll : '',
```

Comment:
Problem is that it complains when we try to destructure a property in snake_case